llama-index-integrations/readers/llama-index-readers-jira/llama_index/readers/jira/base.py

```diff
@@ -20,6 +20,19 @@ class PATauth(TypedDict):
     api_token: str
 
 
+def safe_get(obj, *attrs):
+    """Safely get nested attributes from an object."""
+    try:
+        for attr in attrs:
```

Comment:
does this work for `issue.permalink` ?

Previously we had `issue.permalink()`

Now, we do `safe_get(issue, "permalink")` -- following this code, `if callable(attr)` would be `if callable("permalink")` which would always be false. And due to the try/except, it might be hiding this error

I think the code should something like

```
obj = getattr(obj, attr)
if callable(attr)
  obj = obj()
```